core/primitives/src/version.rs

```diff
@@ -164,7 +162,10 @@ pub enum ProtocolFeature {
     /// alpha is min stake ratio
     #[cfg(feature = "protocol_feature_fix_staking_threshold")]
     FixStakingThreshold,
-    /// Validate account id for function call access keys.
+    /// Charge for contract loading before it happens.
+    #[cfg(feature = "protocol_feature_fix_contract_loading_cost")]
+    FixContractLoadingCost,
+    /// Charge for contract loading before it happens.
```

Comment:
This documentation bug got reintroduced as you probably just used `git revert ...` to create the commit.  Should we fix it?